zaza/openstack/charm_tests/ovn/tests.py

```diff
@@ -139,6 +140,97 @@ def setUpClass(cls):
                 'ovs-vswitchd',
             ]
 
+    def test_conf_enable_hardware_offload(self):
+        """Enable hardware offload and confirm unit configuration.
+
+        Note, this is a shallow non-blackbox test to confirm charm operation
+        only.
+        """
+        if self.current_release <= openstack_utils.get_os_release(
+                'bionic_ussuri'):
+            logging.info('bionic kernel does not have netdevsim, skip')
+            return
```

Comment:
Is there any reason we don't use `self.SkipTest(....)` instead of logging an returning?  I've seen this pattern a few times, and always wondered that.

Reply:
Good question, I'm inclined to update my proposal based on this review feedback.